### ISSUE TITLE
[Warning] This project is likely to be abandoned.

### ISSUE THREAD

**Issue by mrcageman** [nathanreyes/v-calendar#1503]
With the release of Vue 3.5, I recently ran into some issues with this library. I then noticed what a lot of you have probably noticed as well:

The last known activity on this project stems from **October 2023**, which is a full year ago. The last pull request was merged on **September 10, 2023**, which was from the author himself.

After this, there has been no known interaction from the author and pull requests have been left untouched. Those which have been closed were closed by the submitters themselves. There is no response from the author in the issues section. Only community members that are eager to provide solutions – none of which make it into the package itself.

As such, I feel it's quite safe to assume that this project is now **unmaintained** and **abandoned**. 

If anyone has anything to add to this, please add a comment.

---
P.S.

I'm currently looking for viable alternatives to replace this package in my own projects. The ones I've stumbled upon so far include:
- [Radix Vue - Date Picker](https://www.radix-vue.com/components/date-picker)
- [Vue3 Date Picker](https://vue3datepicker.com)

Feel free to add more alternatives if you have any.

**Comment by onlime** [comment]
Thanks @mrcageman for your proposed alternatives. Will probably migrate to [Vue Datepicker](https://vue3datepicker.com/) in the near future. But for now, this fork fixes the main issue with Vue 3.5: https://github.com/nathanreyes/v-calendar/issues/1498#issuecomment-2453589028

cross-posting @tigading's workaround here for the desperate:

> I think this might help you all; switch to using this guy’s forked version.
> https://www.npmjs.com/package/@angelblanco/v-calendar

**Comment by throrin19** [comment]
Unfortunatelly, I think this VCalendar is more simple to use and have more features than others.

Vuetify have his datepicker too but since the publish in final version in his v3, the datepicker lost key features and it is very complex to use....

**Comment by onlime** [comment]
> Unfortunatelly, I think this VCalendar is more simple to use and have more features than others.

right, that's why I am still sticking with VCalendar. So far, the [@angelblanco/v-calendar](https://www.npmjs.com/package/@angelblanco/v-calendar) v3.1.3 fork works perfect under all latest Vue 3.5 versions.

@nathanreyes are you still planning to maintain this wonderful project? We would all very much appreciate!

**Comment by throrin19** [comment]
@onlime Ok, I try [Vue Datepicker](https://vue3datepicker.com/) : It works like a charm. Easy to integrate and customize.

**Comment by Faq** [comment]
Yeah https://github.com/Vuepic/vue-datepicker seems to have all basic implementation. I guess if something is missing, can request some feature.
